UPGRADE.md

```diff
@@ -1,3 +1,39 @@
+# Ilios 3 Upgrade Notes
+
+## General steps
+
+0. Back up your database. _Always._
+
+1. Upgrade you codebase via composer.
+
+ ```bash
+cd YOUR_ILIOS_APPLICATION_ROOT 
+sudo -u apache composer install
+```
+
+2. Execute any pending database migrations.
+
+ ```bash
+cd YOUR_ILIOS_APPLICATION_ROOT
+sudo -u apache bin/console doctrine:migrations:migrate --env=prod --no-interaction
+```
+
+3. Clear your application cache.
+ 
+ ```bash
+cd YOUR_ILIOS_APPLICATION_ROOT
+# clear out the cache dir
+sudo rm -rf var/cache/*
```

Comment:
This shouldn't be necessary in production. Really just a troubleshooting technique.